Kitodo/src/main/java/org/kitodo/production/forms/createprocess/ProcessDataTab.java

```diff
@@ -189,8 +190,12 @@ public void generateProcessTitleAndTiffHeader() {
         List<ProcessDetail> processDetails = this.createProcessForm.getProcessMetadataTab().getProcessDetailsElements();
         Process process = this.createProcessForm.getMainProcess();
         try {
+            StructuralElementViewInterface docTypeView = createProcessForm.getRuleset().getStructuralElementView(
+                docType, createProcessForm.getAcquisitionStage(), createProcessForm.getPriorityList());
+            String processTitle = docTypeView.getProcessTitle()
+                    .orElseThrow(() -> new ProcessGenerationException(docType + " does not declare a process title."));
```

Comment:
same here

Reply:
That means you click on the button and nothing happens. It is a matter of taste. I found it better to get a message than to have an application whose buttons just do not work, but ok, I change it.